openstack/identity_auth_scope_v3.go

```diff
@@ -22,11 +22,37 @@ func flattenIdentityAuthScopeV3Roles(roles []tokens3.Role) []map[string]string {
 	return allRoles
 }
 
-func GetTokenDetails(sc *gophercloud.ServiceClient) (*tokens3.User, *tokens3.Domain, *tokens3.Project, []tokens3.Role, error) {
+func flattenIdentityAuthScopeV3ServiceCatalog(catalog tokens3.ServiceCatalog) []map[string]interface{} {
```

Comment:
can you pass the pointer here?
```suggestion
func flattenIdentityAuthScopeV3ServiceCatalog(catalog *tokens3.ServiceCatalog) []map[string]interface{} {
```

Reply:
Done.